hardhat.config.ts

```diff
@@ -30,13 +30,15 @@ const config: HardhatUserConfig = {
         accountsBalance: "100000000000000000000000000",
       },
       initialBaseFeePerGas: 0,
+      chainId: 31337,
     },
     docker: {
       url: "http://localhost:8545",
       accounts: {
         mnemonic: devMnemonic,
       },
-      chainId: 1337,
+      initialBaseFeePerGas: 0,
+      chainId: 31337,
```

Comment:
ganache uses 1337 and since i swapped out the docker image with hardhat so we can use logging it's now 31337

not sure why I added initialBaseFeePerGas but it matches the `hardhat` configuration in the same config, could be that deployment didnt work without it, I can see if removing it still works if you want

Reply:
ohhh, ok 👍